ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/decoders/GeneralDecoderTest.kt

```diff
@@ -432,4 +433,36 @@ class GeneralDecoderTest {
             , Toml.decodeFromString(test)
         )
     }
+
+    @Serializable
+    data class TimeTable(
```

Comment:
Could you please add the negative scenario? When we were not able to parse the value?
And the date inside the string `a = "1979-05-27T00:32:00-07:00"`

Reply:
yup, will do that